### ISSUE TITLE
Invalid credentials for current docker-compose.yml registry setup

### ISSUE THREAD

**Issue by HDVUCAIR** [quiltdata/quilt#778]
I've been testing a local registry with the current docker-compose.yml file.  However, when it comes time to login with the default admin/quilt login/password, I get an "invalid credentials" error.

I looked around in the code for where the default password might be set, but I haven't found anything yet.  I found mention in the forums of an old environmental variable to be set, but that was from the Django days and no longer appears in the current docker-compose*.yml files.

Thanks,
John.

**Comment by thbeh** [comment]
I am also getting the same "invalid credentials" error. Anyone manage to solve this? Thanks in advance

**Comment by greenspray9** [comment]
Due to lack of documentation on the change, It was hard to to find but I managed to figure this out. I tried the naive way to directly add a new user in postgres container but that didn't work. So after carefully analyzing the Dockerfile, I saw that the `scripts` directory was being copied to the registry container but not being run. So I did a `docker ps`, copied the container id of the registry and `docker exec -it <container-name>` et voila, you see the `scripts`  directory so now just do is`scripts/create_admin.py`.

**TL;DR**
1. `docker ps`, copy the container id of the registry
2. `docker exec -it <container-name>` 
3. `scripts/create_admin.py`

**Comment by meffij** [comment]
The answer above is correct, thank you for your detailed write-up @greenspray9 

For a faster response, you can also join us at `quiltusers.slack.com`

**Comment by thbeh** [comment]
Thanks @greenspray9 . It works

**Comment by greenspray9** [comment]
How can i get access to the slack chat ?

**Comment by ResidentMario** [comment]
@greenspray9: If you follow the link you'll be prompted for an email, if you enter that in you'll automatically get an invite code to that email address. You can click through from there to configure an account, and then sign into the Slack client with that account.

**Comment by greenspray9** [comment]
It says -> "If you have an @quiltdata.io email address, you can create an account." but i don't have quiltdata.io domain email address

**Comment by akarve** [comment]
Sorry about that, here's the right link https://slack.quiltdata.com/

**Comment by meffij** [comment]
Sorry for the bad link, I have updated my original comment with the correct link.

**Comment by kevinemoore** [comment]
The local docker registry instructions are now fixed. See the sub-section on creating an admin account added in #805.
